### PR TITLE
Update UserManager docs for spelling/clarity

### PR DESCRIPTION
**Pull request by devhammer**
Updated /// comments for several method definitions for spelling/clarity.

### CODE REVIEW

File: src/Microsoft.AspNetCore.Identity/UserManager.cs [hunk 1]
```diff
@@ -1859,7 +1859,7 @@ public virtual Task<string> GenerateTwoFactorTokenAsync(TUser user, string token
 
         /// <summary>
         /// Gets the last <see cref="DateTimeOffset"/> a user's last lockout expired, if any.
-        /// Any time in the past should be indicates a user is not locked out.
+        /// Any time value in the past should be indicates a user is not currently locked out.
```
Comment (Rick-Anderson) [inline src/Microsoft.AspNetCore.Identity/UserManager.cs]: A time value in the past indicates a user is not currently locked out.

Comment (HaoK) [inline src/Microsoft.AspNetCore.Identity/UserManager.cs]: @devhammer once you make this last update I can merge this PR

Comment (Rick-Anderson) [reply src/Microsoft.AspNetCore.Identity/UserManager.cs]: @HaoK  can you make that change and merge?

**Comment by HaoK** [comment]
Ported to https://github.com/aspnet/AspNetCore/pull/5952 since the code has moved

**Pull request closed without merging**